test/boost/cdc_test.cc

```diff
@@ -46,9 +46,6 @@ static cql_test_config mk_cdc_test_config() {
     auto ext = std::make_shared<db::extensions>();
```

Comment:
This function should not be necessary at all now.
It was introduced in b130b76274fe7414b01ada9b41554c78540454e1 to speed up tests.
(I don't recall if the slowdown in startup due to enabling CDC wasn't fixed at some point)

Reply:
Sounds great but also deserves a PR on its own ;).